packages/frontend/src/components/post/Post.tsx

```diff
@@ -84,44 +84,17 @@ export default function Post({
         setIsAction(finalAction)
     }, [isMine, finalAction])
 
-    const postInfo = (
-        <div className="space-y-3">
-            <header className="flex items-center gap-4">
-                <Avatar name={epochKey ?? nanoid()} />
-                <span className="text-xs font-medium tracking-wide text-black/80">
-                    {subtitle}
-                </span>
-            </header>
-            <section className="text-sm font-medium tracking-wider text-black/90">
-                {compact ? (
-                    <LinesEllipsis
-                        className="break-words whitespace-break-spaces"
-                        text={content}
-                        maxLine="4"
-                        ellipsis="..."
-                        component="p"
-                    />
-                ) : (
-                    <p className="break-words whitespace-break-spaces">
-                        {content}
-                    </p>
-                )}
-            </section>
-        </div>
-    )
-
     const handleVote = async (voteType: VoteAction) => {
```

Comment:
This component should have a single responsibility to display post data only. I suggest to use another props named `onVote` like L46 `onComment` to handle voting logic outside. And you can clean up unrelated props and state in this component.

Reply:
fixed